db/migrate/20171019201011_migrate_to_new_mentor_industry_model.rb

```diff
@@ -0,0 +1,15 @@
+class MigrateToNewMentorIndustryModel < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
+      if user.mentor == true
+        if user.mentor_industry == "Business"
+          MentorIndustryUser.create(mentor_industry_id: 1, user_id: user.id, career_stage: user.stage_of_career)
```

Comment:
This was why I left the `MentorIndustry.create` comments, because for the three existing mentor industries, I manually put them in the Rails console. Maybe it should be done in the migration instead?

Reply:
Yes, yes it should- so the three existing mentor_industries are created when the MentorIndustry table is created. But it is also in the seed file for development purposes.

Reply:
This one has been resolved- in the `create_mentor_industries` migration, it generates the three mentor industries.